### PR TITLE
CH-35

### PR DESCRIPTION
**Pull request by shibachurro**
Change log:
- app.js restructured
	- refresh token code moved into new client.on()
	- might want to revisit and code a setRefresh.js (included in this commit)
- Added JP to authors in package.json
- Added config.json for prefix "!"
- Added and implemented commands: player.js, roster.js
- Added a new integration test for /commands/player.js in /__tests__/integration/CommandsTest.intTest.js
- Commented potential issue on nbaUtils.js for getDaysPlayedOn, need discussion if further action is needed

### CODE REVIEW

File: __tests__/integration/CommandsTest.intTest.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const nbaUtils = require('../../util/nbaUtils')
+const nbaData = require('../../api/nbaData')
+jest.setTimeout(10000)
+
+describe('test suite for /commands/', () => {
+  describe('tests for player.js', () => {
+    test('check that output can handle null', async () => {
```
Comment (Laturine) [inline __tests__/integration/CommandsTest.intTest.js]: is this test handling null or ensuring that null doesn't get sent back?

Comment (shibachurro) [reply __tests__/integration/CommandsTest.intTest.js]: This test is checking that the return msg should not be null, that said this test might need to be moved under, if it hasn't already, daysPlayedOn()

File: app.js [hunk 2]
```diff
@@ -1,40 +1,35 @@
...
+
+client.once('ready', () => {
+  console.log('READY in apptest.js')
+})
+
+client.on('message', async message => {
+  if (!message.content.startsWith(prefix) || message.author.bot) return
+  const args = message.content.slice(prefix.length).split(/ +/)
+  const command = args.shift().toLowerCase()
+  if (!client.commands.has(command)) return
```
Comment (Laturine) [inline app.js]: If user enters an invalid command either create a !help command and refer to that or list all available commands.

File: __tests__/integration/CommandsTest.intTest.js (hunk 1 above)
Comment (shibachurro) [reply __tests__/integration/CommandsTest.intTest.js]: per @Laturine delete this test, unit test already written, and create ticket to write test for a negative case (ie a case in which null gets through daysPlayedOn() )

**Pull request merged**